### PR TITLE
fix lf timing issue

### PR DESCRIPTION
**Pull request by jcantrill**
Fix LogForwarding timing issue and other minor test fixes

cc @ewolinetz @igor-karpukhin  I discovered an issue of sysctl setting not being applied by the node-tuning-operator but I'm hoping its not a larger issue and just the cluster I stood up.  Bumping the timing here as previously only gave ES 2minutes to come up

We should run this a couple of times if it works

### CODE REVIEW

File: hack/testing/assertions [hunk 1]
```diff
@@ -1,5 +1,5 @@
 #!/bin/bash
-source "$(dirname $0)/utils"
+source "$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )/utils"
```
Comment (ewolinetz) [inline hack/testing/assertions]: is it really necessary to `cd` as part of this `source` statement?

Comment (richm) [reply hack/testing/assertions]: in general, yes, if you want the "real" absolute path

**Comment by igor-karpukhin** [comment]
Log forward test still fails

**Comment by jcantrill** [comment]
all tests blocked by https://github.com/openshift/cluster-node-tuning-operator/pull/96

**Comment by jcantrill** [comment]
In master at least, there was  a change in the node-tuning-operator where the sysctl changes required by ES are not being applied to the node.  waiting on the referenced PR to merge in order to unblock

**Pull request merged**